# My collection of useful stuffs in omics sciences
# Homepage: https://sites.google.com/site/drnguyenphuoclong/
The below list is merely from my scientific experience and perspective. Hope it helps!

# Useful omics tools:

PiGx: Pipelines in Genomics [link]
Network-Based Integrative Analysis (for metabolomics data combined with related transcriptomics information) with MetaBridge [link]
TCGAbiolinks: An R/Bioconductor package for integrative analysis with GDC data [link] [link]
MetProc: Separating Measurement Artifacts from True Metabolites in an Untargeted Metabolomics Experiment [link]
Small Molecule Pathway Database [link]
The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins [link]
FELLA: an R package to enrich metabolomics data [link]
LIPEA: a package for Lipid Pathway Enrichment Analysis [link]
CEU Mass Mediator: a tool for searching metabolites in different databases (Kegg, HMDB, LipidMaps, Metlin, MINE and an in-house library) [link]
MS-DIAL: data-independent MS/MS deconvolution for comprehensive metabolome analysis [link]
Qiita: rapid, web-enabled microbiome meta-analysis [link]
REVIGO [link] and WEGO [link] for GO annotation and visualization.
BioJupies: Automated Generation of Interactive Notebooks for RNA-Seq Data Analysis in the Cloud [link]
LipidPedia - An encyclopedia of lipids providing associated biomedical information [link]
MitoMiner 4.0 - A database of mammalian mitochondrial localisation evidence, phenotypes and diseases [link]
LipiDex: An Integrated Software Package for High-Confidence Lipid Identification [link]
MetaboList: Annotation of Metabolites from Liquid Chromatography-Mass Spectrometry Data [link]

# Good data handling and analysis packages

car: Companion to Applied Regression [link]: I sometimes use this package to reassign the name of the columns in the data set
forcats for handling categorical variables, especially for visualization [link]. E.g., fct_rev(fct_infreq(x)) for "ordered from top to bottom, highest count to lowest".

# Visualization tools:

BPG: Seamless, automated and interactive visualization of scientific data [link]
Biorender - professional visualization for life sciences [link] 
Note: not free for publication!
From data to viz (with code) [link]
DIVE: Turn your data into stories without writing code [link]
# Good books:

Interpretable Machine Learning [link]
Seeing theory - to study probability and statistics [link]
Fundamentals of Data Visualization [link]
Mathematics for Machine Learning - only for beginners or practical users. [link] 
Feature Engineering and Selection: A Practical Approach for Predictive Models [link]
Metabolic profiling series from Springer. Click here for the recent version

# Good papers:

Structure annotation of all mass spectra in untargeted metabolomics [link]
Peak annotation and verification engine (PAVE) for untargeted LC-MS metabolomics [link]
From RNA-seq reads to differential expression results [link]
Evaluation and comparison of bioinformatic tools for the enrichment analysis of metabolomics data [link]
From mass to metabolite in human untargeted metabolomics: recent advances in annotation of metabolites applying liquid chromatography-mass spectrometry data [link]
Bad practices in evaluation methodology relevant to class-imbalanced problems [link]

# Good courses:

Khan academy [link]
DataCamp [link]
MIT Computational biology [link]

# Others:

RNAseq analysis in R [link]
Common transition words and phrases [link]
Cheat Sheets for AI, Neural Networks, Machine Learning, Deep Learning & Big Data [link]
Writing for a Nature journal [link]
The Conversation - evidence-based news [link]
